spec/controllers/reviewers/applications_controller_spec.rb

```diff
@@ -89,6 +89,17 @@
       let(:user) { create :reviewer }
       before { sign_in user }
 
+      context 'given the application has mentor comments' do
+        let(:mentor) { create :mentor }
+        let!(:mentor_comment) { Mentor::Comment.create(user: user, commentable_id: application.id, text: 'something') }
+
+        before { get :show, params: { id: application } }
+
+        it 'assigns @mentor_comments' do
+          expect(assigns :mentor_comments).to contain_exactly(mentor_comment)
```

Comment:
I know we check for `assigns` all over the place but this is discouraged in recent Rails versions. Can you regexp-match for a unique comment string in the `response.body` instead?